pkg/diag/diag_test.go

```diff
@@ -87,3 +96,33 @@ func TestRun(t *testing.T) {
 		})
 	}
 }
+
+func TestRunErr(t *testing.T) {
+	tests := []struct {
+		description string
+		shouldErr   bool
+		labels      map[string]string
+		ns          []string
+		expectedErr error
+	}{
+		{
+			description: "handles error",
+			shouldErr:   true,
+			labels: map[string]string{
+				"skaffold": "session",
+			},
+			ns:          []string{"foo"},
+			expectedErr: fmt.Errorf("following errors occurred someErr\n"),
```

Comment:
nit: Seems our linter is mad w/ the error text:
```
Error: pkg/diag/diag_test.go:115:28: error strings should not be capitalized or end with punctuation or a newline (golint)
```

Fix this and things should be all green

Reply:
thank you for the review, Aaron, should be fixed now!